app/models/comment.php

```diff
@@ -1,19 +1,40 @@
 <?php
 class Comment extends AppModel
 {
+    CONST FIRST_COMMENT   = 1;
+    CONST MIN_BODY_LENGTH = 1;
+    CONST MAX_BODY_LENGTH = 200;
+    CONST TABLE           = 'comment';
+
     public $validation = array(
         'body'       => array(
-            'length' => array('validate_between', 1, 200,),
+            'length' => array('validate_between', self::MIN_BODY_LENGTH, self::MAX_BODY_LENGTH),
         ),
     );
 
-    public function getAll($offset, $limit, $thread_id)
+    public function getAll($offset, $limit, $thread_id, $filter_username)
```

Comment:
Changed po to static. I also changed po the getAttributes function to static so I won't need the instance of comment po.https://github.com/renzosunico/MyClassroom/commit/d5f535adc164f29e8ade125406b6224ca690a44a